closed/autoconf/custom-hook.m4

```diff
@@ -138,6 +138,9 @@ AC_DEFUN_ONCE([OPENJ9_PLATFORM_SETUP],
     elif test "x$OPENJDK_BUILD_OS" = xwindows; then
       OPENJ9_PLATFORM_CODE=wa64
       OPENJ9_BUILDSPEC="win_x86-64_cmprssptrs"
+    elif test "x$OPENJDK_BUILD_OS" = xmacosx; then
+      OPENJ9_PLATFORM_CODE=oa64
```

Comment:
Is there a related pull-request that makes `oa64` meaningful?

Reply:
Its used with OpenJ9.gmk '-tag:define "PLATFORM-$(OPENJ9_PLATFORM_CODE)' as an option to jpp.

Reply:
Yes, I knew that. Is https://github.com/eclipse/openj9/pull/262 how 'oa64' becomes meaningful?

Reply:
My misunderstanding, yes combined with eclipse/openj9#262, and further future changes this platform will generate a valid set of binaries.  The changes are incremental and not meant to be a final solution.

Reply:
I guess the important point, though, is that this won't work without eclipse/openj9#262.